<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configuracion</class>
 <widget class="QDialog" name="Configuracion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuracion</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Recursos.qrc">
    <normaloff>:/Icono</normaloff>:/Icono</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(241, 239, 255);</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>MS Reference Sans Serif</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Servidor:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="InDirecion">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 255, 0);
background-color: rgb(170, 255, 127);</string>
     </property>
     <property name="text">
      <string>http://</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>MS Reference Sans Serif</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Puerto:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="InPuerto">
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 255, 0);
background-color: rgb(170, 255, 127);</string>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>MS Reference Sans Serif</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Habilitar cajas</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="chkCaja1">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 127);
border-color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>Caja1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkCaja2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 127);
border-color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>Caja2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkCaja3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 127);
border-color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>Caja3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkCaja4">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 127);
border-color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>Caja4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnAbrir">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(207, 207, 207);
border-color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Abrir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnGuardar">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(207, 207, 207);
border-color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Guardar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Autor:Mauricio Vega</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
